IPython/terminal/ptutils.py

```diff
@@ -21,6 +21,23 @@
 
 _completion_sentinel = object()
 
+def _ellide(string, *, min_ellide=30):
```

Comment:
It's `elide` with one L.

However, `ellipsis` (plural ellipses) with two Ls is correct. English, such a wonderfully consistent language ;-)